test/export/test_serialize.py

```diff
@@ -213,7 +214,7 @@ def forward(self, x: torch.Tensor, y: torch.Tensor) -> torch.Tensor:
 
 @unittest.skipIf(not torchdynamo.is_dynamo_supported(), "dynamo doesn't support")
 class TestDeserialize(TestCase):
-    def check_graph(self, fn, inputs, dynamic_shapes=None, _check_meta=True) -> None:
+    def check_graph(self, fn, inputs, dynamic_shapes=None, _check_meta=True, inputs_mutated = False) -> None:
```

Comment:
In unittests, we can always assume inputs are mutated, so that we don't need additional flag here.